changelog/1047.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix permission resolution for :meth:`.Thread.permissions_for` to use :attr:`Permissions.send_messages_in_threads` instead of :attr:`Permissions.send_messages` for calculating implicit permissions.
```

Comment:
```suggestion
Fix permission resolution for :class:`Thread`\s to use :attr:`Permissions.send_messages_in_threads` instead of :attr:`Permissions.send_messages` for calculating implicit permissions.
```
since `permissions_for` is used by other methods (like check decorators) as well, this can be generalized a bit

Reply:
shoot, do those need versionchanged as well

Reply:
[`582955c` (#1047)](https://github.com/DisnakeDev/disnake/pull/1047/commits/582955cdabae128581b84848212a93ed64ccd1b9)

Reply:
not really, documenting it in the main location is fine.